pkg/build/controller/controller_test.go

```diff
@@ -390,14 +399,11 @@ func TestHandleBuild(t *testing.T) {
 		if tc.inStatus != buildapi.BuildPhaseError && build.Status.Phase == buildapi.BuildPhaseError && len(build.Status.Message) == 0 {
 			t.Errorf("(%d) errored build should set message: %#v", i, build)
 		}
-		if len(tc.outputSpec) != 0 {
-			build := ctrl.BuildStrategy.(*okStrategy).build
```

Comment:
so here's my conclusion. what we do above with `copy, err := kapi.Scheme.Copy(build); originalBuild := copy.(*buildapi.Build)` is done here as well. the object is reinitialized with okStrategy (okStrategy.build is  *api.Build) that points to original build. and then it's compared to see if the the tc.output.spec.name is different than expected. i didn't get any differences when comparing the build object with okStrategy and the originalBuild.